spec/models/task_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  context 'validations' do
+    it { should validate_presence_of :name }
+    it { should validate_length_of(:name).is_at_most(250) }
+    it { should validate_length_of(:description).is_at_most(2500) }
+
+    describe 'due_at_cannot_be_in_the_past' do
+      let(:task) { build(:task, due_at: 1.day.ago) }
+
+      it 'raises error' do
```

Comment:
Na verdade o `save` não vai disparar um erro, e sim retornar `false` e adicionar os erros em `errors`, certo?

Nesse caso, o que acha de alterar o texto para algo como `validates that due_at cannot be in the past` removendo  o `describe`? Estou sugerindo dessa forma pra ficar parecido com o que o `shoulda-matchers` faz, mas se preferir pode usar outro padrão.